src/PhpSpreadsheet/Spreadsheet.php

```diff
@@ -660,9 +660,10 @@ public function getAllSheets(): array
      */
     public function getSheetByName(string $worksheetName): ?Worksheet
     {
-        $worksheetCount = count($this->workSheetCollection);
-        for ($i = 0; $i < $worksheetCount; ++$i) {
-            if (strcasecmp($this->workSheetCollection[$i]->getTitle(), trim($worksheetName, "'")) === 0) {
+        $trimWorksheetName = trim($worksheetName, "'");
+
+        for ($i = 0; $i < $this->getSheetCount(); ++$i) {
```

Comment:
This absolutely adds overhead by moving a function call from outside the loop to inside. It will definitely not be approved.

Reply:
Reverted in latest commit.